oeps/oep-0002.rst

```diff
@@ -47,7 +47,7 @@ Specification
 Each repo in the Open edX system will include a file ``openedx.yaml``, with the
```

Comment:
(In particular, that job runs the `oep2` script that's in https://github.com/edx/repo-tools.